CHANGELOG.rst

```diff
@@ -2,6 +2,30 @@
 Release Notes
 =============
 
+.. _Release Notes_7.38.2:
+
+7.38.2 / 6.38.2
+======
+
+.. _Release Notes_7.38.2_Prelude:
+
+Prelude
+-------
+
+Release on: 2022-08-09
+
+- Please refer to the `7.38.2 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7382>`_ for the list of changes on the Core Checks
```

Comment:
there's no mention of `7.38.2` in the integrations-core changelog, should there be one?

Reply:
In https://github.com/DataDog/integrations-core/pull/12711/files there is a changelog update so I assume it will be an entry there. So we can keep this link on our side.